total_station_beacon_static_transform_publisher:
  ros__parameters:
    ts_prism_x_offset: '6.9098' # meters
    ts_prism_y_offset: '4.8026' # meters
    ts_prism_z_offset: '2.17185' # meters
    ts_prism_z_rotation: '-0.005047857' #radians